examples/03_qaoa_on_qpus.ipynb

```diff
@@ -22,6 +22,16 @@
   {
```

Comment:
This whole notebook has to be cleaned up (and probably tweaked a bit!)

Reply:
Cleaned in commit [65d8a23](https://github.com/entropicalabs/openqaoa/pull/179/commits/65d8a23c3952390606cfd9937ed6fdca3fb9b41b)